keps/sig-storage/1710-selinux-relabeling/README.md

```diff
@@ -407,18 +568,24 @@ Due to change of Kubernetes behavior, we will implement the feature only for cas
 #### Phase 1
 - Implement the feature only for volumes that are backed by PersistentVolumeClaims with `ReadWriteOncePod` access mode.
   Such volumes can be used only in a single pod and two pods can't ever conflict when using it.
-- Collect metrics of how many other pods would fail because they use a RWO/RWX volume that's used by a pod with different SELinux context on the same node.
-  TBD: create Info level alert ("please consider re-architecting your app not to share volumes this way and/or report it to sig-storage")?
-This phase can go Beta (be enabled by default) or even GA without breaking anything.
+- Collect metrics of how many other pods would fail because they use a RWO/RWX volume that's used by a pod with different SELinux label on the same node.
+
+This phase went Beta (enabled by default) in Kubernetes 1.28, without `SELinuxChangePolicy` field in PodSpec!
+
+#### Phase 1.5
+
+- Introduce `SELinuxChangePolicy` field, under `SELinuxChangePolicy` feature gate.
+  - In this phase, the only allowed value is `Recursive`.
+  - The field is not interpreted, it's used only by cluster admins to proactively opt-out from `SELinuxMount`
+  - We need to graduate it to Beta / enabled by default before enabling `SELinuxMount` feature gate.
```

Comment:
So while `SELinuxChangePolicy` is enabled, what is the default in phase 2 (or phase 1.5)? I guess it defaults to `null` which has fluidity in its meaning. And are we planning to change the default in next release after this is enabled?

Reply:
`null` still means `MountOption`, but the whole field is not used by kubelet, only by metrics.